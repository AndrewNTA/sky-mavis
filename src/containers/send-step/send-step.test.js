import React from 'react';
import renderer from 'react-test-renderer';
import SendStep from './index';

jest.mock('react-router-dom', () => ({
  useNavigate: () => jest.fn(),
}));
jest.mock('react-redux', () => ({
  useDispatch: () => jest.fn(),
  useSelector: () => jest.fn(),
}));
test('SendStep should render send step page', () => {
  const component = renderer.create(
    <SendStep />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});