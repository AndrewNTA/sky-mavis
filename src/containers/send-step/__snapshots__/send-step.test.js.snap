// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SendStep should render send step page 1`] = `
<div
  className="st-container"
>
  <div
    className="success-modal"
  >
    <div
      className="sm-overlay"
    />
    <div
      className="sm-main"
    >
      <div
        className="sm-title"
      >
        Successfully sent
      </div>
      <div
        className="sm-content"
      >
        <div>
          <span>
            Your
          </span>
          <strong
            className="sm-currency"
          >
            
          </strong>
          <span>
            has been sent!
          </span>
        </div>
        <div>
          Thank you for using our service
        </div>
      </div>
      <button
        className="r-primary-btn full"
        onClick={[Function]}
      >
        OK
      </button>
    </div>
  </div>
  <div
    className="st-header"
  >
    <img
      alt="back"
      className="st-back-icon"
      onClick={[Function]}
      src="back-icon.png"
    />
    <span
      className="st-header-title"
    >
      Send Assets
    </span>
  </div>
  <div
    className="st-form"
  >
    <div
      className="st-form-row"
    >
      <div
        className="st-form-label"
      >
        from
      </div>
      <div
        className="r-text-input-container readonly"
      >
        <div
          className="r-text-input-readonly"
        >
          My wallet (7300...3334)
        </div>
      </div>
    </div>
    <div
      className="st-form-row"
    >
      <div
        className="st-form-label"
      >
        to
      </div>
      <div
        className="r-text-input-container false"
      >
        <input
          className="r-text-input"
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      className="st-form-row"
    >
      <div
        className="st-form-label"
      >
        asset
      </div>
      <div
        className="r-asset-input-container"
      >
        <div
          className="r-asset-info"
        />
        <img
          alt="options"
          className="r-asset-option-icon"
          onClick={[Function]}
          src="option-icon.png"
        />
      </div>
    </div>
    <div
      className="st-form-row"
    >
      <div
        className="st-group-label"
      >
        <div
          className="st-form-label"
        >
          amount
        </div>
        <div
          className="st-group-label-right"
        >
          <span>
            available:
          </span>
          <span>
            
          </span>
        </div>
      </div>
      <div
        className="r-text-input-container false"
      >
        <input
          className="r-text-input"
          onChange={[Function]}
          type="text"
          value=""
        />
        <div
          className="st-max-icon"
          onClick={[Function]}
        >
          Max
        </div>
      </div>
    </div>
  </div>
  <div
    className="st-control-group"
  >
    <button
      className="r-secondary-btn medium"
      onClick={[Function]}
    >
      Cancel
    </button>
    <button
      className="r-primary-btn medium"
      onClick={[Function]}
    >
      Send
    </button>
  </div>
</div>
`;
